interface_frontend.py

```diff
@@ -265,6 +265,14 @@ def draw_interface(interface_state, window):
         if card_index is not None:
             draw_card(coordinate, interface_state.dealt_cards[card_index])
 
+    # Blocked cards
+    if interface_state.blocked_cards:
+        cislo = len(interface_state.blocked_cards)
+        blocked_cards_coordinates = program_coordinates[-cislo:]
```

Comment:
Tyto dva radky se taky daji v pohode prepsat do jednoho radku a nemusis zavadet promennou cesky ;).

Reply:
To jsem zapomnela předělat  😂